draft-ietf-ppm-dap.md

```diff
@@ -543,11 +543,24 @@ specific to the given query type. A query is defined as follows:
 ~~~
 opaque BatchID[32];
 
+enum {
+  by-batch-id(0),
```

Comment:
the current draft unified enum cases using underscore.

Reply:
(here and below)

Reply:
@branlwyd I think this hasn't actually been resolved yet? Basically the ask is: change "-" to "_" in enum variants